content/player/blockchain-integration/buying-nfts.md

```diff
@@ -60,9 +60,11 @@ function createOrder(
 )
 ```
 
-Once an Order has been created on-chain, any user with sufficient MANA, and as long as the Order is not expired, can execute the Order and purchase the NFT.
+Once an Order has been created on-chain, the asset will be listed for sale so that any user can buy it. 
 
-An order is executed by calling the `executeOrder` function in the Marketplace Smart Contract.
+When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by signing a transaction in MetaMask or the user's Wallet. Users can set an allowance for the amount of the asset they want to buy or a higher cap to continue using the Marketplace without changing it every time. We recommend using the Allowance cap by default.
```

Comment:
Following that this is a technical document, we should either pause at:
```
When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first.
```
To include the signature of the transaction that is done against the MANA contract to change the allowance or have a section to describe how the allowance is performed.

Reply:
```suggestion
When a user wants to buy an asset listed in the Marketplace, they have to allow the Marketplace Smart Contract to operate with their MANA first. This is done by sending a transaction. Users can set an allowance for the amount of the asset they want to buy or a higher cap to continue using the Marketplace without changing it every time. We recommend using the Allowance cap by default.
```